pypowsybl_jupyter/explorer.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from IPython.display import display
+from pypowsybl.network import Network
+from pypowsybl_jupyter.svg import display_svg
+
+import ipywidgets as widgets
+
+
+def network_explorer(network: Network):
+    """
+    Creates a basic explorer widget for this network.
+    """
+
+    vl_input = widgets.Text(
+        value='',
+        placeholder='Voltage level ID',
+        description='Voltage level:',
+        disabled=False,
+        continuous_update=True
+    )
+
+    found = widgets.SelectMultiple(
+        options=[],
+        value=[],
```

Comment:
options can be initialized with all the voltage levels.

value can be omitted, but maybe it's not the usual practice?

```suggestion
        options = network.get_voltage_levels(attributes=[]).index,
```